build-logic/buildquality/src/main/kotlin/gradlebuild/incubation/tasks/IncubatingApiReportTask.kt

```diff
@@ -13,9 +13,8 @@ import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.TaskAction
-import org.gradle.kotlin.dsl.submit
+import org.gradle.kotlin.dsl.*
```

Comment:
:x: let's remove `*` imports please.

Reply:
Added a comment that this workarounds https://youtrack.jetbrains.com/issue/KTIJ-24390/